packages/js/test/plugins/auctionHouseModule/cancelBid.test.ts

```diff
@@ -0,0 +1,171 @@
+import { Keypair } from '@solana/web3.js';
+import test, { Test } from 'tape';
+import { sol } from '@/types';
+import {
+  metaplex,
+  killStuckProcess,
+  createNft,
+  assertThrows,
+  createWallet,
+} from '../../helpers';
+import { createAuctionHouse } from './helpers';
+import { Bid } from '@/plugins/auctionHouseModule';
+
+killStuckProcess();
+
+test('[auctionHouseModule] cancel a Private Bid on an Auction House', async (t: Test) => {
+  // Given we have an Auction House and an NFT.
+  const mx = await metaplex();
+
+  const nft = await createNft(mx);
+  const { client } = await createAuctionHouse(mx);
+
+  // And we put a private bid on that NFT for 1 SOL.
+  const { bid } = await client
+    .bid({
+      mintAccount: nft.address,
+      tokenAccount: nft.token.address,
+      price: sol(1),
+    })
+    .run();
+
+  t.false(bid.canceledAt);
+
+  // When we cancel the given bid.
+  const { bid: canceledBid } = await client.cancelBid({ bid }).run();
```

Comment:
Here we won't return `bid` anymore.

```suggestion
  await client.cancelBid({ bid }).run();
```

This means the test has to then fetch the bid using `findBidByAddress` and make assertions on that.

Same for all other tests.